examples/http/src/main/java/io/opentelemetry/example/http/HttpClient.java

```diff
@@ -49,7 +49,7 @@ public void set(HttpURLConnection carrier, String key, String value) {
         }
       };
 
-  private void initTracer() {
+  private static void initTracer() {
```

Comment:
Can we also change this to `initTracerSdk()`, since we're not really initializing a tracer here.